third_party/llvm-project/compiler-rt/METADATA

```diff
@@ -1,21 +1,26 @@
 name: "compiler-rt"
 description:
-  "Filtered subtree at third_party/llvm-project/compiler-rt."
+  "Subdirectory subtree of llvm-project."
 
 third_party {
-  url {
-    type: LOCAL_SOURCE
-    value: "https://cobalt.googlesource.com/third_party/llvm-project/compiler-rt_filtered_mirror"
+  identifier {
+    type: "ChromiumVersion"
+    value: "114.0.5735.351"
```

Comment:
This seems wrong—
Chromium doesn't include this tree at all, does it?

Reply:
updated